.chloggen/O11Y-3152.yaml

```
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: prometheusremotewriteexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add otel_scope_info metric to prometheus remote write exporter.
```

```suggestion
note: Emit `otel_scope_info` metric and add `otel_scope_name` and `otel_scope_version` labels to metrics
```
I think we should also call out that the scope labels will be added to metrics. I did some wordsmithing here as I was trying to add that info, feel free to word this how you would like.